Rakefile

```diff
@@ -1,38 +1,34 @@
-require 'rubygems'
+# frozen_string_literal: true
```

Comment:
if this remains our fork and is disconnected from upstream, we might use our default Rakefile and have the yard stuff sit in a .yardopts